src/sirocco/parsing/_yaml_data_models.py

```diff
@@ -416,7 +418,13 @@ class ConfigAvailableData(ConfigBaseData):
 
 
 class ConfigGeneratedData(ConfigBaseData):
-    pass
+    @field_validator("computer")
+    @classmethod
+    def invalid_field(cls, value: str | None) -> str | None:
```

Comment:
```suggestion
    def invalid_field(cls, value: str | None) -> None:
```

This never returns anything but `None`.